File: Anul II/Medii de proiectare si programare/Laborator/lab08/C#/Ubb.BikeContest.Networking/Protocol/Response/AllParticipantsResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ubb.BikeContest.Model;

namespace Ubb.BikeContest.Networking.Protocol.Response
{
    [Serializable]
    public class AllParticipantsResponse : IResponse
    {
        private readonly List<Participant> participants;

        public AllParticipantsResponse(List<Participant> participants)
        {
            this.participants = participants;
        }

        public virtual List<Participant> Participants { get { return participants; } }
    }
}

File: Anul II/Medii de proiectare si programare/Laborator/lab11/java/networking/src/main/java/com/ubb/protocol/response/AllParticipantsResponse.java
package com.ubb.protocol.response;

import com.ubb.model.Participant;

import java.util.Collection;

public class AllParticipantsResponse implements Response {
    private final Collection<Participant> participants;

    public AllParticipantsResponse(Collection<Participant> participants) {
        this.participants = participants;
    }

    public Collection<Participant> getParticipants() {
        return participants;
    }
}
